<launch>
    <!-- ros2 bag play _2022-04-07-14-14-35_robotarium/_2022-04-07-14-14-35_robotarium.db3 
    -\-rate 1.0 
    -\-clock 
    -\-start-paused 
    -\-remap /stereo_camera/left/image_raw:=/image_raw /odometry/filtered:=/odom 
    -\-topics /stereo_camera/left/image_raw /odometry/filtered -->

    <node
        pkg="ratslam"
        exec="ratslam_pc"
        name="ratslam_pose_cells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="robotarium" /> <!-- fix topic root -->
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/neoslam_robotarium.yaml" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_em"
        name="ratslam_experience_map"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="use_sim_time" value="true" />
        <param name="topic_root" value="robotarium" /> <!-- fix topic root -->
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
        <param from="$(find-pkg-share ratslam)/config/neoslam_robotarium.yaml" />
    </node>      
</launch>

